testsuite/testcases/src/public_fullnode_performance.rs

```diff
@@ -31,18 +32,25 @@ pub struct PFNPerformance {
     num_pfns: u64,
     add_cpu_chaos: bool,
     add_network_emulation: bool,
+    config_override_fn: Option<OverrideNodeConfigFn>,
     shuffle_rng_seed: [u8; 32],
 }
 
 impl PFNPerformance {
-    pub fn new(num_pfns: u64, add_cpu_chaos: bool, add_network_emulation: bool) -> Self {
```

Comment:
Let's revert this change before landing.

Reply:
I moved the new `pfn_spam_duplicates` test to #13019